e2e/core/ui/CommonActions.js

```diff
@@ -16,7 +16,7 @@ class CommonActions {
      */
     static operate(element) {
```

Comment:
Should be in this common file methods from load page file, like goUrl and getTitlePage